std_msgs/Header header
uint8[4]        robotHealth
uint8[4]        remain_bullet
CellStatus[6]   cellState
RobotPose[4]    robotPose
float64         gameTime
